firebase-functions/src/main/java/com/google/firebase/functions/FunctionsComponent.java

```diff
@@ -0,0 +1,78 @@
+// Copyright 2018 Google LLC
```

Comment:
```suggestion
// Copyright 2022 Google LLC
```

Reply:
oops, looks like auto merge does not take these comments into account